advanced/list-of-log-names.md

```diff
@@ -4,43 +4,43 @@ description: On this page all the available logs are listed
 
 # List of log names
 
-Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names(which can be used in place of individual log names, see [configuring channels](../admin-commands/config/configuring-channels.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates.
+Logs, categorized by what they do are listed here, and are **case sensitive**. Individual logs are written in _UPPER\_CASE_, while category names (which can be used in place of individual log names, see [_**configuring channels**_](../management-commands/config.md)) are _Capitalized_. Each one of them is separated by categories in the lists below. If you want to log everything, use an asterisk to include all things that can be logged, including ones that will be introduced in future updates.
 
 {% hint style="warning" %}
-Some logs require the bot to have the "View Audit Logs" permission enabled.
+Some logs require the bot to have the "View Audit Log" permission enabled.
 {% endhint %}
 
 ## Moderation
 
-This category is used for actions by moderators or ones that moderators could use. Most of them are fired when a command is run unless they start with AUDIT_. In that case, actions that are done with or without using the bot are logged.
+This category is used for actions by moderators or ones that moderators could use. Most of them are fired when a command is run unless they start with AUDIT_, which uses the values in the server's audit log. In that case, actions that are done with or without using the bot are logged.
 
 | Name              | Emitted when     |
 | ----------------- | ---------------- |
-| BAN               | Someone gets banned |
-| INFRACTION_DELETE | Someone removes an infraction|
-| KICK              | Someone gets kicked |
-| MUTE              | Someone gets muted |
-| PAUSE_ROLE_ADD    | An expiring role gets added  |
-| PAUSE_ROLE_EXTEND | A role expiration date gets extended |
-| PAUSE_ROLE_REMOVE | A role gets temporarily removed |
+| BAN               | A user gets banned |
+| INFRACTION_DELETE | A moderator removes an infraction|
```

Comment:
Deletes is more accurate than removes, links the pages to the relevant (sub)commands to take the actions.